### PR TITLE
release 0.9.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+## Summary
+
+This is an awesome release!
+
+### Bugfixes
+
+  * Monkey patch ls_files until ruby-git/ruby-git#320 is resolved
+  * Reraise exception rather than exit so we can rescue a derived StandardError when using skip_broken option
+
+### Enhancement
```
Comment (alexjfisher) [inline CHANGELOG.md]: Pluralise?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 2018-02-15
+
+## Summary
+
+This is an awesome release!
```
Comment (alexjfisher) [inline CHANGELOG.md]: Aren't all releases? :)

Comment (alexjfisher) [reply CHANGELOG.md]: @ekohl Is https://github.com/voxpupuli/modulesync/pull/142 the most significant change?  I really think we should have something more sensible the 'this is an awesome release!'

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md]: I just wanted to be funny :P

**Pull request merged**